### DISCUSSION TITLE
"Future of MusE"

### DISCUSSION THREAD

**Discussion by kybos** [muse-sequencer/muse#1218] [Team Posts]
I asked the guy for details and here is his response:

> Hi,
> 
> Yeah, like a roadmap. I see MusE as a rough diamond, a rock waiting to be cut and processed. The software capability is amazing; what is missing in my opinion are some tweaks and brushes to make MusE to apple _[kybos: I guess he means "appeal"]_ to the mass. I can help with that. My day job is around e-commerce and digital marketing. I know how to create brands and how to talk to the right audience. With the right branding/marketing approach MusE will gain followers, users and even developers if you want. That's where I can help you.
> 
> As I said in the email, if we share the same goals, I would be happy to help you guys. That's why I asked about your roadmap.
> 
> Let me know your thoughts and if this is something that you are looking for.
> 
> Cheers

It would be quite a coincidence but I can imagine it's the same person that is behind the "Muse group" I discovered yesterday.

**Comment by spamatica** [comment]
Okay, this is actually getting a bit wierd..

But I'm not sure I buy that it could be the same people. I don't see the motivation since we are currently Linux only. They clearly want a bigger audience and it must be a big risk if they expect MusE to become multiplatform before they can promote it. But who knows, maybe that is what he is looking for in the roadmap?

In any case, what does this kind of proposal mean for us, do we want help with marketing?

I'm personally rather old-school open source activist and get a little scared when it gets too shiny. But that is just me, and I'm sure I can be convinced if it looks trustworthy.

What are your thoughts?

**Comment by kybos** [comment]
I'm skeptical too. Why should anybody want to invest into a niche Linux app like MusE? 
I don't know the business, so I don't get the implications of this all.

I would prefer a transparent offer from him: Put your cards on the table, explain your motivation, and if we all agree, the roadmap (and "the future") can be planned accordingly. Especially as I already wrote him that we've just got out a new release and are only just starting to discuss the roadmap for the next one...

**Comment by terminator356** [comment]
Was not aware of the following, I only knew score was big-time from day 1, it was a huge hit
 that looked professional and felt a bit out of my 'hacker' league: From WP:
> The MuseScore company uses income from their commercial sheet music sharing service
 to support the development of the free notation software.
In 2018, the MuseScore company was acquired by Ultimate Guitar, which added full-time paid developers to the open source team. In April 2021, it was announced that a parent company, Muse Group, would be formed to support MuseScore, Ultimate Guitar, and other acquired properties (including Audacity)

Wonder what role Werner plays these days.

Crystal ball: They a) add audio or b) try to fuse MusE into it for audio and/or our fine piano rolls + drums.
Hence the mysterious contact? Hey finally, after 20 years, it's a MusE reunion! Super MusE!

>  I see MusE as a rough diamond, a rock waiting to be cut and processed.

"I'm gonna make you a star, baby."
Cut and processed. Methodically. Clinically. Ruthlessly. Until there's no soul left.
Relax, I'm just making some humour of it. Not saying I'm against it one way or the other at the moment.

> Why should anybody want to invest into a niche Linux app like MusE?

Because it's freakin' good. Well, not too shabby. There, I said it. I can say it. I'm confident of
 a lot of that hard work that we do. You know my viewpoint, if I could get paid that would be nice.
One legitimate reason: I have so many branches and things I was supposed to finish.
But time is limited ATM. If I could devote full time to coding again I could get a lot done.

What if something far different happens instead?
"We own this app now. You're all gone. Change your name or get lost."
"Ominous", eh?

**Comment by kybos** [comment]
"Rough diamonds" sometimes break when processed...

But back to the point: I/We have to answer the mail, being polite people. 

I see 2 options:

1. I do it (put you on cc). As mentioned before, I want to know what it's all about in clear and concrete terms. I suspect the guy has clear expectations, so let the cat out of the bag.
2. You (one of you) takes over and handles it. His email: oldcastlehq@protonmail.com

Your vote?

**Comment by spamatica** [comment]
I'm okay with me not replying, if I put it like that :D 
Should we make it publicly on the forum or continue on mail?

My personal suggestion is to outline a few features we want/plan to add (like we were talking about earlier) but make no big promises, it's a roadmap... And ask what he wants to work on more specifically, and what his expectations are...

**Comment by terminator356** [comment]
Since he contacted kybos I guess let him respond.
I see no reason to carry on the discussion publicly in a forum.

Back in 8 hours...

**Comment by kybos** [comment]
He is already complaining on the forum that we "don't want help". Seems to be very impatient...
I will write the mail anyway, probably this evening. I also don't think the public forum is the right place for this kind of discussion.

**Comment by terminator356** [comment]
Thanks kybos.
Seems I'm getting excited over a random marketing dude posting what amounts to spam?
So it's all just a coincidence, not related to the muse group? Ha.

**Comment by kybos** [comment]
I suppose you saw the response from this person in your mail box.
Feel free to answer him/her. I think I now know what I wanted to know (and suspected). I guess you have more experience in handling such things.

**Comment by terminator356** [comment]
A lot of fair points made. He genuinely wants to help it succeed, from a new user viewpoint.

New users are always on my mind when I code. I try to make it as pleasant as I can.
It makes me sad and embarrassed when they are turned away from good software
 that maybe has an obscure bug that just happens to show up right as they begin to test it,
 causing them to immediately dismiss the program and move on. Sad.

As you can guess I tend to agree with the funding support thing he mentions.
I raise the topic from time to time but I won't push it. Most big projects are like that these days.
A few of the years I have worked on this app have actually been full-time at my own cost,
 when I should have been out looking for a job. I was able to coast along with part-time work
 and savings and so on. I guess I just really like this app. A lot. ;-)
I was thinking of approaching some local music stores to see if they would be interested in sponsoring.
Then there's GitHub sponsors. We'd have a much better shot if we presented ourselves as a 
 coherent group rather than just me alone.
Otherwise we remain this organic 'move at our own pace' thing, with not much time to do
 all the things he mentions, although kybos has been doing an amazing job moving it forward.

I suppose one question is "what's in it for him?".
Does it involve paying him? Does he expect to make money from helping?
Is that reasonable if we go from zero to hero with his help?

Thanks guys. Talk at ya tomorrow...

**Comment by spamatica** [comment]
As you probably saw I replied to the mail conversation. To me it seems he gave a lot of thought to his ideas so we should discuss it at least.
I must admit I never gave much thought to the idea of sponsorship and I'm not exactly sure who would be interested but it is somewhat intriguing to me. Maybe there are dots that can be connected with little effort?

Tim, I know you have worked _a lot_ on MusE but I didn't know it was exclusively at times. :-o If you managed to get by I guess I should not complain :). I do see how making it easier to have some income from MusE would be a really good thing though.
What is necessary to create a github sponsoring for the team, a bank account?

Regarding what is in it for him. I could of course be wrong but it seems to me he just wants MusE to succeed.. 
Put another way, if he is expecting to get paid this will simply not work out. 

If (big IF) we end up in a few years that MusE actually has an income, then we can discuss how to divide the money :)

**Comment by kybos** [comment]
There is certainly nothing wrong in discussing these things.
I'm sure it would be highly beneficial for MusE if Tim were able to work at it full-time. If you see a chance to raise some money for this purpose, well, then go for it. 
I personally am not in a position to be able to do a full-time development for MusE, even if there were some money available.

Back to the "future": I am pretty skeptical about the latest mail from the guy. IMO it clearly shows that he has no idea about software development, UX design and even MusE itself. At least I don't believe that it would be feasible and even advisable to try to transform MusE (which was shaped after Cubase, a professional tool for experts) into a toy for beginners which know absolutely nothing about audio and midi. I guess there would be better candidates for that (like LMMS).
But that's just my opinion/impression, don't let yourselves be put off by it ;-).

**Comment by spamatica** [comment]
Den sön 9 maj 2021 kl 10:05 skrev kybos ***@***.***>:

> There is certainly nothing wrong in discussing these things.
> I'm sure it would be highly beneficial for MusE if Tim were able to work
> at it full-time. If you see a chance to raise some money for this purpose,
> well, then go for it.
> I personally am not in a position to be able to do a full-time development
> for MusE, even if there were some money available.
>
> Back to the "future": I am pretty skeptical about the latest mail from the
> guy. IMO it clearly shows that he has no idea about software development,
> UX design and even MusE itself. At least I don't believe that it would be
> feasible and even advisable to try to transform MusE (which was shaped
> after Cubase, a professional tool for experts) into a toy for beginners
> which know absolutely nothing about audio and midi. I guess there would be
> better candidates for that (like LMMS).
> But that's just my opinion/impression, don't let yourselves be put off by
> it ;-).
>

And it is very good that you mention that. We are just three people and
what binds us together, MusE, should not be mistreated by changing it into
something we do not want ourselves.
I didn't look at the videos that he mentioned yet but judging by the colors
in the thumbnails you are probably right, we are aiming for slightly more
mature users ;)

Even though I am sure he has a point that people like shiny things that are
easy to understand (my translation). And I think we can learn from that,
but best to keep this development slow and organic. ...even if I feel we
are very lucky that you showed up and were able to transform the UI in a
pretty short amount of time!

/Robert

**Comment by kybos** [comment]
> Even though I am sure he has a point that people like shiny things that are
> easy to understand (my translation).

Of course they do, I have tried to stress this point many times in our discussions ;-). 
But it's not easy to make a complex application simple to use (especially not post-factum). That's the Holy Grail of UX design, and even many commercial apps fail at it more or less.

He is probably also right about the rebranding. MusE is not "cool", the icon/logo is not "cool" either. Still I am not sure if it's a good idea...
